src/core/lib/surface/channel_init.h

```diff
@@ -74,8 +73,7 @@ class ChannelInit {
   /// Construct a channel stack of some sort: see channel_stack.h for details
   /// \a type is the type of channel stack to create
```

Comment:
This line can be removed, since the parameter is no longer present.

Reply:
fixed